lessons/Early-Math/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+- Are there any numbers on this line that don't have an opposite?
+** _Students may guess that zero does not have an opposite - however zero is its own opposite!_
+
+=== Investigate
+
+Explain to students that the function used to represent "the opposite" is called `negate`. It can be used like any other operator that we put at the top of a Circle of Evaluation - with one important difference: when we use `negate`, we can include just one argument inside the Circle.
+
+[.lesson-instruction]
+--
+- Take a look at the Circle below. What does it evaluate to?
+** _The Circle below evaluates to @math{-2}._
+
+[.image]
+@show{(coe '(negate 2))}
+
+- What happens if we nest a negating Circle _inside of_ another negating Circle?
+** _The opposite of the opposite of a number is the number itself._
+
+[.image]
+@show{(coe '(negate (negate 2)))}
+
+- Practice applying `negate` on @printable-exercise{true-or-false-negate.adoc}. Evaluate each side of the equation to confirm if it is true or false.
+- Explore Circles with Variables, and review previously explored properties (Identity, Commutative, Associative, and Distributive) on @printable-exercise{true-or-false-negate-2.adoc}.
+--
+
+When we want to write a mathematical expression that indicates that negating is happen, we just use a negative sign! So, while we can read @math{-45} by simply saying "negative 45," another way to read that value's name is by saying, "the opposite of 45."
+
+
+[.strategy-box, cols="1", grid="none", stripes="none"]
+|===
+|
+@span{.title}{Pedagogy Note: Can we call it "minus 45"?!}
+Precise language is crucial when dealing with this symbol: `-`.
+
+In math we want students to read for _comprehension_, not just mechanically sounding out each symbol in order. We read the expression @math{12 - 9} as "12 minus 9" __because subtraction is the "verb"__, not because the minus sign happens to be in the middle!
+
+When we encounter a value such as @math{-20}, however, it is confusing and misleading to name that value "minus 5." Subtraction is not happening here - negation is! But if the teacher uses the "symbols in order" wording, it reinforces the lower level of reading comprehension for students.
+|===
+
+
+
+=== Synthesize
+
+- Two opposites are 8 units apart from each other. What are they?
+** @math{4} and @math{-4}
+- Two opposites are 42.5 units apart from each other. What are they?
+** @math{21.5} and @math{-21.5}
+- Cia says that opposites are always negative. Are they right? Why?
+** _No, opposites are not always negative. When we negate a negative value, we get a positive value. Additionally, zero is its own opposite, and it is neither positive nor negative._
+
+== Absolute Value @duration{30 minutes}
+
+=== Overview
+
+Students consider the meaning of absolute value, and apply the concept to Circles of Evaluation using `num-abs`.
+
+=== Launch
+
+[.lesson-instruction]
+What is the distance between these two points on the number line: @math{-18} and @math{5}?
```

Comment:
We have asked them to draw a number line of the integers from -10 to 10. Either pick two integers within that range or change the directions for which numbers should be on the numberline that students and teachers draw.